app-doc/info-manual/metadata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>gnu-emacs@gentoo.org</email>
+    <name>Gentoo GNU Emacs project</name>
+  </maintainer>
+  <longdescription>
+    The GNU Project distributes most of its manuals in the Info format.
+    There are two primary Info readers: The stand-alone info program
+    (<pkg>sys-apps/texinfo</pkg>) and the info package in GNU Emacs
+    (<pkg>app-editors/emacs</pkg> or <pkg>app-editors/xemacs</pkg>).
+
+    This manual describes how to get around inside of Info, various
+    advanced Info commands, and contains references to other sources,
+    which explain how to generate Info files from Texinfo files.
```

Comment:
I'd probably mention the motivation behind extracting this (both packages rely on it for the info tutorial), but otherwise LGTM

Reply:
I've also added a blocker against `app-editors/emacs`, otherwise the entry would be listed twice in the Info directory.